vllm/engine/llm_engine.py

```diff
@@ -284,6 +284,12 @@ def _stop_sequences(self, seq_groups: List[SequenceGroup]) -> None:
                 if stopped:
                     continue
 
+                # Check if the sequence has reached max_sequence_len.
+                if seq.get_len() == \
```

Comment:
Please use `()` for line continuation: https://google.github.io/styleguide/pyguide.html#32-line-length

Reply:
Nit:
```suggestion
                if seq.get_len() >= \
```